src/styles/todo.scss

```diff
@@ -100,3 +109,14 @@
     opacity: 0.5;
   }
 }
+
+.todo__edit-field {
```

Comment:
The classname 'todo__edit-field' should follow the BEM naming convention. Consider renaming it to 'todo__edit-field-container' or 'todo__edit-field-wrapper'.